src/components/QuestionEdit/index.tsx

```diff
@@ -216,16 +255,15 @@ export default function QuestionEdit({
       </form>
       <div aria-hidden={!isPreview} className={classNames(styles.container, { [styles.toggledOff]: !isPreview })}>
         <Question
-          text={questionText}
-          options={questionOptions}
-          preSelectedChoice={questionCorrectOptionId}
+          selectedQuestion={getValues()}
           onClose={onClose}
-          isAttempted
-          isCorrect
           isPreview
           isQuestionSaved={isQuestionSaved}
-          submitResponse={() => handleSubmit()}
+          submitResponse={() => onFormSubmit(getValues())}
```

Comment:
Don't we need the handleSubmit validation checks here?

Reply:
no.. added validation rules in FormInput component. react-form-hook will do the validations

Reply:
Yeah but few other validations are done in the handleSubmit function right?

Reply:
Yeah, then don't we have to perform those validations here too?

Reply:
validations which are applied on FormInput I added them as validation rule. 
and validation which are not applied on FormInput, we have to validate on form submit. it's not necessory too keep all validations at one place

Reply:
Did not understand your comment.

Reply:
I'm saying no need to perform those validations here as `react-hook-form` will not submit the form until validations are satisfied.